src/components/TaskView.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useContext } from 'react'
-import { Box, Stack, Text, useToast } from '@sanity/ui'
+import { Box, Heading, Stack, useToast } from '@sanity/ui'
```

Comment:
```suggestion
import { Box, Text, Stack, useToast } from '@sanity/ui'
```
See above comments on standards in `Heading` vs `Text`